### PR TITLE
Add missing changelog for 1.0.0

### PR DESCRIPTION
**Pull request by AlMcKinlay**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0.0 (2017-02-15)
+
+* Added support for pk7s (need to pass gen=7 option to parser)
+* Renamed to pkparse
+* Added generation 7 data (pokemon, moves, abilities, games and items)
+* idNo is now either a 5 digit tid for gen 6, or 6 digit tid for gen 7
```
Comment (not-an-aardvark) [inline CHANGELOG.md:8]: We should also probably mention `fullTid` somewhere, and the fact that `tid` and `sid` are now deprecated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0.0 (2017-02-15)
+
+* Added support for pk7s (need to pass gen=7 option to parser)
```
Comment (not-an-aardvark) [inline CHANGELOG.md:5]: Can you also update the documentation for this function in the readme?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+* Renamed to pkparse
+* Added generation 7 data (pokemon, moves, abilities, games and items)
+* idNo is now either a 5 digit tid for gen 6, or 6 digit tid for gen 7
+* Will use semver from now on (breaking changes only on major version releases)
+
```
Comment (not-an-aardvark) [inline CHANGELOG.md]: If I recall correctly, there were some other breaking changes too (e.g. relating to how ribbon data is stored). We should document all the breaking changes.

**Review by not-an-aardvark — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (AlMcKinlay) [reply CHANGELOG.md]: The ribbon changes were reverted. I can't think of any breaking changes, but I'll see if @SnowPhoenix can think of any.

File: CHANGELOG.md (hunk 1 above)
Comment (AlMcKinlay) [reply CHANGELOG.md:8]: Yeah, good plan.

File: CHANGELOG.md (hunk 2 above)
Comment (AlMcKinlay) [reply CHANGELOG.md:5]: I already did that. Unless there's something specific you think I've missed.

Comment (not-an-aardvark) [reply CHANGELOG.md:5]: You're right, never mind. (Now that I think about it we might also want to document `pkparse.SUPPORTED_GENS`, but that's probably less important.)

Comment (AlMcKinlay) [reply CHANGELOG.md:5]: Done :-)

**Review by not-an-aardvark — APPROVED** [review]

**Pull request merged**